### ISSUE TITLE
how does it work with multiple cookers  ?

### ISSUE THREAD

**Issue by licheng5625** [syssi/xiaomi_cooker#33]
I saw the way to call cooking start/stop. There is no place to point to specific the entity id , so I assume it does not work with multiple  cookers ?

**Comment by syssi** [comment]
You are right. The service implementation is too basic and must be improved.

**Comment by tieppm** [comment]
@licheng5625  you user model cocker xiaomi?

**Comment by lawsongsong** [comment]
请问 @syssi 怎么设备多设备，我有两个电饭煲。。。
